figwheel-bridge.js

```diff
@@ -16,7 +16,7 @@ var config = {
 var React = require('react');
 var createReactClass = require('create-react-class');
 var ReactNative = require('react-native');
-var WebSocket = require('WebSocket');
+//var WebSocket = WebSocket; //require('WebSocket');
```

Comment:
could you elaborate please?  how will it work

Reply:
It is a global var from now on, I will just remove this line

Reply:
fixed